public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -24,3 +37,57 @@ export async function fetchTenants(http: HttpStart): Promise<DataObject<Tenant>>
 export async function fetchTenantNameList(http: HttpStart): Promise<string[]> {
   return Object.keys(await fetchTenants(http));
 }
+
+export function transformTenantData(rawTenantData: any) {
+  const tenantList = map(rawTenantData, (v: any, k: string) => ({
+    tenant: k.startsWith('global') ? GLOBAL_USER_DICT.Label : k,
```

Comment:
Wondering why use `startsWith` rather than exact match? If user creates some tenant like `global_sales`, that will be renamed to `Global`, right?

Reply:
You are right. Corrected and tested.